client-misk/src/main/kotlin/app/cash/backfila/client/misk/embedded/Backfila.kt

```diff
@@ -9,29 +10,45 @@ import okio.ByteString
  * the Backfila dashboard UI.
  */
 interface Backfila {
-  fun <Type : Backfill<*, *>> createDryRun(
+  fun <Type : Backfill<*, *, *>> createDryRun(
     backfill: KClass<Type>,
     parameters: Map<String, ByteString>,
     rangeStart: String?,
     rangeEnd: String?
   ): BackfillRun<Type>
 
-  fun <Type : Backfill<*, *>> createWetRun(
+  fun <Type : Backfill<*, *, *>> createWetRun(
     backfillType: KClass<Type>,
     parameters: Map<String, ByteString>,
     rangeStart: String?,
     rangeEnd: String?
   ): BackfillRun<Type>
 }
 
-inline fun <reified Type : Backfill<*, *>> Backfila.createDryRun(
-  parameters: Map<String, ByteString> = mapOf(),
+inline fun <reified Type : Backfill<*, *, *>> Backfila.createDryRun(
+  parameters: Any? = null,
```

Comment:
might not be worth using reified here, just use normal class type argument so you get the inference for Param :\

Reply:
Yeah that was my first reaction too. Then I started thinking that it really didn't matter what the underlying class was as long as it turns into a Bytestring map that represents what backfila will send. I'm open to this changing though.

Reply:
mainly I want the type there so when you write the test intellij tells you what type to provide. I can feel the questions coming already.. "what do you put there?"

Reply:
yeah... very true